core/templates/dev/head/pages/creator-dashboard-page/creator-dashboard-page.controller.spec.ts

```diff
@@ -115,6 +115,7 @@ describe('Creator dashboard controller', function() {
           AlertsService: null,
           CreatorDashboardBackendApiService: CreatorDashboardBackendApiService
         });
+        ctrl.$onInit();
```

Comment:
Explanation for this change: https://www.codelord.net/2017/01/09/unit-testing-angular-components-with-%24componentcontroller/

Reply:
Probably should include it as an inline comment, here and elsewhere?

Reply:
Done.